build_runner/lib/src/entrypoint/options.dart

```diff
@@ -649,6 +702,15 @@ class _CleanCommand extends Command<int> {
   }
 }
 
+Set<String> _buildDirsFromOutputMap(Map<String, String> outputMap) {
+  var dirs = new Set<String>();
+  outputMap.forEach((k, v) {
```

Comment:
we aren't using `k` here so we could use a for loop around `outputMap.values`. Even better we could express this as what it is rather than how we get it.

```dart
Set<String> _buildDirsFromOutputMap(Map<String, String> outputMap) =>
    outputMap.values.where((v) => v != null).toSet();
```